### DISCUSSION TITLE
Enable discussion on this repo?

### DISCUSSION THREAD

**Discussion by Haiz14** [fastapi/full-stack-fastapi-template#1106] [Questions] [answered]
[Enabling instructions (GitHub.com)](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/enabling-or-disabling-github-discussions-for-a-repository)

It would be way simpler to seperate discussions regarding the project and issues regarding this project.

e.g. I have a query with regards to the file [celeryworker_pre_start](https://github.com/tiangolo/full-stack-fastapi-template/blob/master/backend/app/celeryworker_pre_start.py)

Like why is there celery in this project? There's no celery container in the docker-compose.

There's also no info on exactly why celery is used in this repo?

**Comment by Haiz14** [comment]
Celery was recently closed. https://github.com/tiangolo/full-stack-fastapi-template/pull/694

**Comment by tiangolo** [comment, marked as answer]
Thanks! Yes, I just enabled Discussions and we set up the question templates now.

**Comment by DGRYZER** [comment]
Yes, I just enabled Discussions and we set up the question templates now.
